includes/class-tyche-plugin-deactivation.php

```diff
@@ -192,14 +192,14 @@ public function tyche_plugin_deactivation_submit_action() {
 					'headers'     => array( 'user-agent' => 'TSTracker/' . md5( esc_url( home_url( '/' ) ) ) . ';' ),
 					'body'        => wp_json_encode(
 						array(
-							'action'         => 'plugin-deactivation',
-							'plugin_slug'    => $this->plugin_short_name,
-							'url'            => home_url(),
-							'email'          => apply_filters( 'ts_tracker_admin_email', get_option( 'admin_email' ) ),
-							'plugin_name'    => $this->plugin_name,
-							'reason_id'      => isset( $_POST['reason_id'] ) ? wp_unslash( $_POST['reason_id'] ) : '',
-							'reason_text'    => isset( $_POST['reason_text'] ) ? wp_unslash( $_POST['reason_text'] ) : '',
-							'reason_info'    => isset( $_POST['reason_info'] ) ? wp_unslash( $_POST[''] ) : 'reason_info',
+							'action'      => 'plugin-deactivation',
```

Comment:
@nidhitatosaniya Please create a PR and commit these changes on the PLugin Deactivation repo: https://github.com/TycheSoftwares/tyche-plugin-deactivation